### PR TITLE
Add hooks. Do whatever you like.

### PR DESCRIPTION
**Pull request by Zempashi**

### CODE REVIEW

File: playbooks/01_site.yml [hunk 1]
```diff
@@ -78,25 +133,50 @@
       vars:
         kubeadm_pkgs: false
         node_pkgs: true
+  tasks:
+    - include_role:
+        name: hooks_call
+      vars:
+        kubeadm_hook_name: pre_nodes_upgrade
```
Comment (Tassatux) [inline playbooks/01_site.yml:140]:
```suggestion
        kubeadm_hook_name: post_nodes_upgrade
```

**Comment by Zempashi** [comment]
@Tassatux : I rename some hooks:

* nodes => for all cp and workers
* workers => for only workers

File: docs/hooks.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+It comes with great possibilities, hence use with caution is required.
+
+### Using hooks
+
+Hooks are discovered in `kubeadm_hooks_dir` which default to `{{ inventory_dir }}`
+Simply create a `kubeadm.<hook_name>.d` directory and puts tasks yaml files in it.
+
+Note: Those are *tasks* like in roles not *playbooks*.
+
+you can register you're `tasks` files by using the variable `kubeadm_hooks_<hook_name>`.
```
Comment (b-abadie) [inline docs/hooks.md:11]: you're -> your

**Review by Tassatux — APPROVED** [review]
LGTM :)

**Pull request merged**